spec/channels/application_cable_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationCable do
```

Comment:
was attempting to test application cable but might need to connect it to the front end to accomplish this